content/docs/_index.md

```diff
@@ -32,6 +32,13 @@ layout: single
 - [Named Dates](named_dates/) {{< label >}}2.4.0{{< /label >}}
 - [Duration Values](durations/) {{< label >}}2.4.0{{< /label >}}
 
+## Manual Pages
+
+- [task(1)](man/task.1)
```

Comment:
FYI: Hugo will create those pages as `/man/task.1/`, causing a redirection to it when `/man/task.1` is called.
You can use the `check-links` script from the `bin` directory to check this.
E.g. start the page locally and run
```
check-links --no-extern
```